### ISSUE TITLE
MX16, Sardi 9.6.0 - some icons' pictures look buggy.

### ISSUE THREAD

**Issue by asqwerth** [erikdubois/Sardi#26]
Hi, trying out Sardi v.9.6 with the new MX Linux icons, on my MX16 installation.   Discovered a problem with quite a few of the icons, where the picture part of the icon doesn't render well and bleeds into the circular field behind the picture.  Also, in Thunar the navigation icons and the "x"  close-tab icons don't render well.

See pic:
![mx16-sardi](https://user-images.githubusercontent.com/22598625/36486856-86035b6a-175a-11e8-912c-285a350c7161.png)

Other details: I'm running compiz, but I don't think that should affect the icons in any way.  No such problem with other icon sets like faenza, papirus, moka, or even your Surfn icons from 2016.

**Comment by ghost** [comment]
thanks for reporting
will check it out

**Comment by ghost** [comment]
strange they look good on ArchMerge
will have to install mxlinux to see that

**Comment by asqwerth** [comment]
MX16 is Debian Jessie-based, rather than Stretch (MX17).  I'm thinking that should have no bearing on icons (unlike gtk3 themes), but just in case, I'm letting you know.

**Comment by ghost** [comment]
thanks it should not
only tested on archmerge and linux mint 18.3 cinnamon
last version now on sourceforge
nothing else right!!

**Comment by asqwerth** [comment]
> nothing else right!!

More info:
I have used various versions of Sardi icons for the last few years (ie, v. 9.3+, 8+, 6+ and even earlier), and did not have this issue on MX14 to 16 with your older versions. 

On the other hand, Sardi v9.6 was fine on a Gnome 3 desktop running an Arch linux derivative.  I wonder if there is some conflict with XFCE 4.12 (the gtk2 version) ?  

I can try testing v9.6 on MX17 and another distro using XFCE 4.12, but may take a few more days as I won't have much time on my home PC right now.

**Comment by asqwerth** [comment]
I was able to reproduce this problem with Sardi 9.6.0 on another installation of MX16 (Jessie).  Again, an older version of Sardi (version 6.1.7 according to log.txt) has been working fine on it.

However, on 2 different installs of MX17 (Stretch) and Manjaro XFCE (Arch variant), there was no problem with Sardi 9.6.0.

Was there some change in how svg graphics are drawn?

**Comment by erikdubois** [comment]
I am not sure.
I had problems with Sardi Ghost Flexible too. Only for Arch distro's. Maybe
we are missing an application or a library?

Puzzling.

Cheers
Erik

2018-02-23 10:43 GMT+01:00 asqwerth <notifications@github.com>:

**Comment by asqwerth** [comment]
Unfortunately I have no knowledge in this area.  For now I'm sticking with ver 9.3.8 (7/12/2017) of Sardi which works fine in all distros.  

Maybe you can try to trace the differences between those 2 versions, because in the changelog for 9.6.0 (16/2/2018), you said that there was a "spring clean of all svg code".

**Comment by asqwerth** [comment]
Hi, I saw your post on MX Linux's G+ page about whether Sardi or Surfn needs an update.  Thank you for asking.  I just downloaded your latest Sardi (9.6.2) and Surfn (last commit Feb 2018) to test.

Sardi 9.6.2 works fine in MX17 (fully updated to ver 17.1) like before, but still has the same rendering bugs in MX16 posted in this issue page.

However, Surfn (25/2/18) works fine in both MX17 and MX16, plus it recognises all the MX apps icons (nice!).

I remember you mentioned somewhere that the indexing or folder structure for Surfn and Sardi are different.  Could that be the reason for how come they behave differently in MX16 (Debian Jessie, gtk3.14)?

Hope that helps.
